src/librustc/middle/stability.rs

```diff
@@ -12,6 +12,7 @@
 //! propagating default levels lexically from parent to children ast nodes.
 
 pub use self::StabilityLevel::*;
+use self::AnnotationKind::*;
```

Comment:
New enums stylistically use namespacing to avoid importing the variants (e.g. `Foo::Bar` is more idiomatic than `FooBar`), perhaps that could be done for this new enum as well?